src/auth/index.ts

```diff
@@ -21,4 +33,28 @@ const setupAuth = password => {
   }
 }
 
-export { setupAuth, isCorrectCredentials }
+// This will be changed into setupAuth at a PR later
+const setupAccessToken = (req, res, next): void => {
+  const { adminPass } = req.cookies
+  const { access = '', authorization = '' } = req.headers
+
```

Comment:
Since an admin is also a user, the standard practice is to make user object:
```
req.user = {
  isAdmin: true/false,
  isLoggedIn: true/false // This is to prevent future accidents when other devs accidentally set req.user = 'undefined' or something
}
```